interbotix_common_toolbox/interbotix_moveit_interface/interbotix_moveit_interface/move_arm.py

```diff
@@ -0,0 +1,358 @@
+        moveit_node_name: str = 'moveit_py',
+        args=None,
+    ) -> None:
+        """
+        Construct the Standalone Interbotix Manipulator module.
+
+        :param robot_model: Interbotix Arm/Cobot model (ex. 'wx200' or 'dx400')
+        :param group_name: (optional) joint group name that contains the 'arm' joints as defined in
+            the 'motor_config' yaml file; typically, this is 'interbotix_arm'
+        :param gripper_name: (optional) name of the gripper joint as defined in the 'motor_config'
+            yaml file; typically, this is 'interbotix_gripper'
+        :param moving_time: (optional) time [s] it should take for all joints in the arm to
+            complete one move
+        :param accel_time: (optional) time [s] it should take for all joints in the arm to
+            accelerate/decelerate to/from max speed
+        :param gripper_pressure: (optional) fraction from 0 - 1 where '0' means the gripper
+            operates at `gripper_pressure_lower_limit` and '1' means the gripper operates at
+            `gripper_pressure_upper_limit`
+        :param gripper_pressure_lower_limit: (optional) lowest 'effort' that should be applied to
+            the gripper if `gripper_pressure` is set to 0; it should be high enough to open/close
+            the gripper (~150 PWM or ~400 mA current)
+        :param gripper_pressure_upper_limit: (optional) largest 'effort' that should be applied to
+            the gripper if `gripper_pressure` is set to 1; it should be low enough that the motor
+            doesn't 'overload' when gripping an object for a few seconds (~350 PWM or ~900 mA)
+        :param topic_joint_states: (optional) the specific JointState topic output by the xs_sdk
+            node
+        :param logging_level: (optional) rclpy logging severity level. Can be DEBUG, INFO, WARN,
+            ERROR, or FATAL. defaults to INFO
+        :param moveit_node_name: (optional) name to give to the core started by this class,
+        defaults to 'moveit_py'
+        """
+        rclpy.init()
+
+        self.arm = InterbotixArmXSInterface(
+            robot_model=robot_model,
+            group_name=group_name,
+            moveit_node_name=moveit_node_name,
+            moving_time=moving_time,
+            accel_time=accel_time,
+
+        )
+        if gripper_name is not None:
+            self.gripper = InterbotixGripperXSInterface(
+                robot_model=robot_model,
+                gripper_name=gripper_name,
+                moveit_node_name=moveit_node_name,
+                gripper_pressure=gripper_pressure,
+                gripper_pressure_lower_limit=gripper_pressure_lower_limit,
+                gripper_pressure_upper_limit=gripper_pressure_upper_limit,
+            )
+
+    def shutdown(self) -> None:
+        """Destroy the node and shut down all threads and processes."""
+        rclpy.shutdown()
+
+
+class InterbotixArmXSInterface:
```

Comment:
Would name this and `InterbotixGripperXSInterface` differently so they don't conflict with the original classes. Maybe something like `InterbotixMoveItArmXSInterface`. Not sure about how verbose that is but